web-services/query/src/main/java/datawave/webservice/query/runner/RunningQuery.java

```diff
@@ -39,6 +26,18 @@
 import org.apache.log4j.Logger;
 import org.jboss.logging.NDC;
 
+import java.security.Principal;
```

Comment:
Just so you know, every time you reorganize imports as such it inevitably causes me to have to resolve conflicts when merging up.